Added fonts?

I'm trying to import some .otf fonts in my sass files and it gives me this error:

ERROR in ../css-loader!../sass-loader!/Users/tonynguyen/Desktop/codetony/src/styles/main.scss
Module not found: Error: Cannot resolve 'file' or 'directory' ../../assets/fonts/BebasNeue.otf in /Users/tonynguyen/Desktop/codetony/src/styles
 @ ../css-loader!../sass-loader!/Users/tonynguyen/Desktop/codetony/src/styles/main.scss 6:19924-19967

I'm sure it's the correct path. however may I ask if we have this in the loader to support different types of font files?

--- a/package.json
+++ b/package.json
@@ -30,6 +30,7 @@
     "chalk": "^1.1.1",
     "css-loader": "^0.23.1",
     "eslint": "^2.3.0",
+    "file-loader": "^0.8.5",
     "history": "^1.13.0",
     "html-webpack-plugin": "^1.6.2",
     "json-loader": "^0.5.4",

